charts/vineyard-operator/README.md

```diff
@@ -0,0 +1,35 @@
+# vineyard operator charts
+
+[![Artifact HUB](https://img.shields.io/endpoint?url=https://artifacthub.io/badge/repository/vineyard)](https://artifacthub.io/packages/helm/vineyard/vineyard-operator)
+
+A helm chart for [vineyard operator][3], which manages all relavant compenents about vineyard.
+
+## Install
+
+Vineyard operator has been integrated with [Helm](https://helm.sh/). Deploy it as follows:
+
+```console
+helm repo add vineyard https://vineyard.oss-ap-southeast-1.aliyuncs.com/charts/
+helm install vineyard-operator vineyard/vineyard-operator
+```
+
+## Uninstall
+
+The installed charts can be removed with
+
+```console
+helm uninstall vineyard-operator
+```
+
+More information about the helm chart could be found at [artifacthub][1] and [parameters][2].
```

Comment:
Add a section about how to create a vineyard cluster after the vineyard operator ready here.

Reply:
Copy.